.ci/install.sh

```diff
@@ -12,6 +12,28 @@ else
   exit 1
 fi
 
+# Parse Flags
+POSITIONAL=()
```

Comment:
consider adding it to a function, so it would be ordered:

```bash
# at top, all the defaults

INSTALL_DIR=$HOME/.fnm

parse_args() {
  while [[ $# -gt 0 ]]; do
    key=$1
    case $key in
      --install-dir)
        $INSTALL_DIR=$2
        shift
        shift
      ;;
    esac
  done
}

# then afterwards

parse_args $@ # (or is it `$*`?)
```

Reply:
This way the argument parsing will _override_ the default definitions (that are much more readable this way, IMO)

Reply:
updated